content/momentum/changelog/4/4-5-0.md

```diff
@@ -0,0 +1,79 @@
+|SD-2863|Story|Fix High "Overflow in uncontrolled allocation size"
+|SD-2864|Story|Fix High "Time-of-check time-of-use filesystem race condition"
+|SD-2866|Story|Fix High "Not enough memory allocated for array of pointer type" alert
+|SD-2871|Story|Fix Error "Inconsistent nullness check"
+|SD-2873|Story|Fix Error " Dead code due to goto or break statement" alert
+|SD-2875|Story|Security Exception Warning "Comparison result is always the same" alert
+|SD-2876|Story|Fix Warning "Ambiguously signed bit-field member" alert
+|SD-3354|Story|ET servers running exploitable nginx in MTA2
+|SD-3615|Story|Improve templating language documentation
+|SD-3619|Story|Fix crashes in Sieve when creating new messages
+|SD-2302|Bug|bounce address localpart HMAC validation is not case insensitive
+|SD-2935|Bug|Crash when outbound_throttle_messages is configured at the same time as core_mailq_throttle_message hook
+|SD-2995|Bug|soft_bounce functionality periodically tries using an invalid connection handle causing a crash
+|SD-3167|Bug|Stack overflow in modules/delivery/http2/http2clnt.c http_send
+|SD-3259|Bug|Momentum incorrectly interprets checkpoint file data on restart
+|SD-3263|Bug|DKIM Oversigning to Avoid Replay Attacks
+|SD-3288|Bug|Transmission API does not sanity check recipient list for empty array
+|SD-3335|Bug|lu_pull not working on AL2 nodes
+|SD-3374|Bug|lu_pull not working on AL2 nodes, round 2
+|SD-3411|Bug|gstack_ecelerity quiet mode does not work on AL2
+|SD-3497|Bug|DSNs are not Generated when RCPTO is Upper Case
+|SD-3535|Bug|extract_sender_from_msg moved to msys namespace
+|SD-3606|Bug|rcpt_to incorrect for "Unconfigured sending domain" rejection events for multi-recipient SMTP injections
+|SD-2751|Story|Adjust threshold for triggering automated memory/thread stats collection in panic log
+|SD-2764|Story|HTTP endpoints to retrieve queue sizes
+|SD-2846|Story|Fix Critical "Non-constant format string" alerts in Momentum
+|SD-2847|Story|Fix Critical "Use of potentially dangerous function" alerts in Momentum
+|SD-2848|Story|Fix Critical "Unbounded write" alerts in Momentum
+|SD-2849|Story|Fix Critical "Potentially overrunning write" alerts in Momentum
+|SD-2850|Story|Fix critical "Static array access may cause overflow" alerts in Momentum
+|SD-2851|Story|Fix Critical "Potentially unsafe use of strcat" alerts in Momentum
+|SD-2855|Story|Fix High "Use of a broken or risky cryptographic algorithm" alert in Momentum
+|SD-2856|Story|Fix High "Multiplication result converted to larger type" alert
+|SD-2857|Story|Fix High "Call to alloca in a loop" alert
+|SD-2858|Story|Fix High "Potentially uninitialized local variable" alerts
+|SD-2867|Story|Fix High "Array offset used before range check" alert in Momentum
+|SD-2868|Story|Fix High  "Incorrect 'not' operator usage" alert
+|SD-2870|Story|Fix Error "Duplicate include guard" alert in Momentum
+|SD-2872|Story|Fix Warning " Unsigned comparison to zero" alert
```

Comment:
As I said in the channel, I want to put all of these together and label them CodeQL